pkg/limits/ingest_limits.go

```diff
@@ -59,6 +69,21 @@ func NewIngestLimits(cfg Config, logger log.Logger, reg prometheus.Registerer) (
 		cfg:      cfg,
 		logger:   logger,
 		metadata: make(map[string]map[uint64]int64),
+		tenantCurrentRecordedStreams: promauto.With(reg).NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: constants.Loki,
+			Name:      "ingest_limits_recorded_streams",
+			Help:      "The current number of recorded streams per tenant.",
+		}, []string{"tenant"}),
+		tenantStreamEvictionsTotal: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Namespace: constants.Loki,
+			Name:      "ingest_limits_stream_evictions_total",
+			Help:      "The total number of streams evicted due to age per tenant.",
```

Comment:
I added a suggestion above to make it clear this is not a global total, due to tenant streams being sharded over a number of pods (i.e. the user should `sum`). I think it might be valuable to add a similar explanation for these other two metrics as well.